app/build.gradle.kts

```diff
@@ -0,0 +1,113 @@
+plugins {
+    alias(libs.plugins.android.application)
+    alias(libs.plugins.kotlin.android)
+    id(libs.plugins.kotlin.parcelize.get().pluginId)
+    id(libs.plugins.navigation.safe.args.get().pluginId)
+    alias(libs.plugins.detekt)
+}
+
+android {
```

Comment:
Can we extract the `compileSdk`, `minSdk`, `targetSdk`, `versionCode` & `versionName` to a standalone gradle file ?